### PR TITLE
Add binary executable scripts

### PR DESCRIPTION
**Pull request by karimcitoh**
@ChrisTerBeke Can you review this PR?

### CODE REVIEW

**Comment by ChrisTerBeke** [comment]
Looks good, but I'm missing the references to these scripts in package.json. Otherwise NPM won't know that these commands need to be added globally.

**Comment by karimcitoh** [comment]
Thanks, great feedback! I will add them.

**Comment by ChrisTerBeke** [comment]
The other script also needs to be copied, that's actually used on The Element. The formide-pi is only used on PI.

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   "author": "Chris ter Beke <chris@printr.com>",
   "license": "GPL-2.0",
   "scripts": {
+    "start": "NODE_ENV=development node app",
```
Comment (ChrisTerBeke) [inline package.json]: This can be removed, the new client has mores specific cases. `npm start` is also not used by us anywhere.

**Comment by karimcitoh** [comment]
The old client only has:
```
"bin": {
    "formide-client": "bin/formide-pi"
  }
```

Can I add the scripts, then, this way?
```
"bin": {
    "formide-pi": "bin/formide-pi",
    "formide-client": "bin/formide-client"
  }
```

**Comment by ChrisTerBeke** [comment]
Yes

**Pull request merged**